pkg/strategy/drift/strategy.go

```diff
@@ -980,6 +977,8 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 		reply.Message(buffer.String())
 	})
 
+	bbgo.RegisterModifier(s)
```

Comment:
multiple strategy instances will override the previous command registration.

Reply:
right now only two options:
1. tg vs strategy = 1 to 1 mapping
2. add this to coreInteract

since we currently have only drift applying this configuration, before any useful mechanism been discussed, I prefer to  keep it simple here and recommend users to try 1.